test/lib/ansible_test/_data/sanity/validate-modules/validate_modules/schema.py

```diff
@@ -249,6 +290,12 @@ def version_added(v, error_code='version-added-invalid', accept_historical=False
                 try:
                     version = SemanticVersion()
                     version.parse(version_added)
+                    if version.major != 0 and version.patch != 0:
```

Comment:
```suggestion
                    if version.major != 0 and version.patch != 0:
```
1. Shouldn't version.minor be here?
2. If the current version is, for example, 1.2.3 and we plan to add add a feature in 1.3.3, will it work correctly? I guess .major and .minor version can be both not `0`

Reply:
Also by the spec, 0.x.y can do whatever it wants. Only for x.y.z with x != 0, new features cannot happen for z != 0.